File: C++/Conditions/R_Age_in_Days.cpp
#include<bits/stdc++.h>
using namespace std;
#define endl '\n'
int main() {
    int n; cin >> n;
    int x = n / 365;
    int a = n - (365 * x);
    int y = a / 30;
    int b = a - (y * 30);
    cout << x << " years" << endl << y << " months" << endl << b << " days";
    return 0;
}

File: Python/R_Age_in_Days.py
totaldays = int(input())

years = totaldays // 365
months = 0
days = 0

months = (totaldays - (years * 365)) // 30

if (years * 365) + (months * 30) != totaldays:
    days = totaldays - ((years * 365) + (months * 30))

print(f"{years} years")
print(f"{months} months")
print(f"{days} days")
